eip/EIPS/eip-4337.md

```diff
@@ -312,6 +313,19 @@ Exceptions to the forbidden opcodes:
 
 ### Reputation scoring and throttling/banning for paymasters
 
+#### Rationale
+
+Paymasters might influence many UserOperations in the mempool.
+To prevent abuse, we "penalize" a paymaster that causes invalidation of large number of UserOperations in the mempool.
+To prevent paymasters from "sybil-attack", we require a paymaster to stake with the system, and thus make such DoS attack very expensive.
+Note that this stake is never slashed, and can be withdrawn any time (after unstake delay)
+
+The stake value is not enforced on-chain, but specifically by each node while simulating a transaction.
+The stake is expected to be above PAYMASTER_STAKE_VALUE (1000 USD in native token)
```

Comment:
See comment above re 1000 USD.  We could have a suggested value but it shouldn't be pegged to USD in an automated way.  The number (denominated in native tokens) should remain steady except during extreme situations.

Reply:
redefined the table with actual native token values